### PR TITLE
View Course as Student, CRUD Announcements (Sensei)

### PR DESCRIPTION
**Pull request by rawrrencee**
@y1bs I am PR-ing this early because of the View Courses page that has the Add to Cart button. You can work on that part first!

I am still working on the "View Course as a student who already bought the course" page. This page is important because it is necessary for your "Approve Course Request" etc pages. Will try to get it up by tomorrow!!

**PLEASE YARN INSTALL AGAIN!**

# Feature
C3 Courses

# Changelog:
See commit log

## Checklist:

- [ ] Merged latest develop
- [ ] PR title makes sense

## Screenshots (where relevant):
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: src/components/Sensei/Course/Create/Announcements.js [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+          onSubmit={e => e.preventDefault()}
+          onFinish={onAddAnnouncement}
+          onFinishFailed={onFinishFailed}
+        >
+          <div className="row">
+            <div className="col-12">
+              <Form.Item
+                name="addAnnouncementTitle"
+                label="Title"
+                rules={[{ required: true, message: 'Please input your notification title.' }]}
```
Comment (natalieloke) [inline src/components/Sensei/Course/Create/Announcements.js]: can i check at the UI level if Senseis only post 'Notification' and not 'Announcement'? understand that backend level it is 'Announcement'

File: src/components/Sensei/Course/Create/Announcements.js [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+            (a, b) => new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime(),
+          ),
+        )
+      }
+    }
+  }
+
+  const removeAnnouncement = async record => {
+    const result = await deleteAnnouncement(record.announcementId)
+    console.log('delAnnouncement', result)
```
Comment (natalieloke) [inline src/components/Sensei/Course/Create/Announcements.js]: some debugging console logs are still here, can remove in future if not needed anymore

File: src/components/Sensei/Course/Create/Announcements.js (hunk 1 above)
Comment (rawrrencee) [reply src/components/Sensei/Course/Create/Announcements.js]: you are right, I was blur and typed notification when I meant announcement!! thought I modified all alr but apparently not haaha thanks!! it is just supposed to be announcement heh

File: src/components/Sensei/Course/Create/Announcements.js (hunk 2 above)
Comment (rawrrencee) [reply src/components/Sensei/Course/Create/Announcements.js]: thanks for catching, will remove!

**Comment by rawrrencee** [comment]
Wah i realised I should put a lot more strings into constants T_T its so mafan tho HAAHA

**Review by natalieloke — APPROVED** [review]

**Review by y1bs — APPROVED** [review]
Looks good!

**Comment by y1bs** [comment]
> Wah i realised I should put a lot more strings into constants T_T its so mafan tho HAAHA

AHAHAH i saw, but ok la, can be reused mah. I think i might refactor some of my code with your constants later

**Pull request merged**